src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -120,6 +126,8 @@ internal static ErrorCode GetErrorCode(this LanguageVersion version)
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7;
                 case LanguageVersion.CSharp7_1:
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7_1;
+                case LanguageVersion.CSharp7_2:
+                    return ErrorCode.ERR_FeatureNotAvailableInVersion7_2;
```

Comment:
Dug up the telemetry comment (from Cyrus): https://github.com/dotnet/roslyn/pull/17435#discussion_r103529003

@CyrusNajmabadi Could you expand on why separate error codes and diagnostics is preferable for LanguageVersion, rather than one parameterized set?